schema_source/aws_serverless_api.py

```diff
@@ -77,6 +77,7 @@ class LambdaTokenAuthorizer(BaseModel):
     FunctionInvokeRole: Optional[str] = lambdatokenauthorizer("FunctionInvokeRole")
     FunctionPayloadType: Optional[Literal["TOKEN"]] = lambdatokenauthorizer("FunctionPayloadType")
     Identity: Optional[LambdaTokenAuthorizerIdentity] = lambdatokenauthorizer("Identity")
+    DisableDefaultFunctionPermissions: Optional[bool]  # TODO Add docs
```

Comment:
What's the rationale for having this as an authorizer property? Is it specific to an authorizer config (more than to a function)?

Reply:
Yea the property specific to the LambdaTokenAuthorizer and the LambdaRequestAuthorizer only since they're the ones that create the Lambda permissions resource